docs/api/specs/desktopAgentCommunicationProtocol.md

```diff
@@ -0,0 +1,457 @@
+---
+id: desktopAgentCommunicationProtocol
+sidebar_label: Desktop Agent Communication Protocol 
+title: Desktop Agent Communication Protocol  (next)
```

Comment:
Superfluous extra space before "(next)".  I'm sure this does no harm, as with any markdown / markup.  But if you have a bit of OCD, it will make you feel dirty and keep you awake at night :)